### DISCUSSION TITLE
Datatable CellEdit cannot get focus in EditMode after Tab.

### DISCUSSION THREAD

**Discussion by YogeshMKumbhar** [primefaces/community#2704] [PrimeFaces] [answered]
This feature was already present in Primefaces 12. If `dataTable` is in `cellEdit` mode, on pressing tab button after editing first cell, focus was shifting to next editable cell with active focus. However, from Primefaces 13 onwards, if you press tab, then cell opens in edit mode, but, focus is somewhere else. I have to again press tab key to make it focusable. The same problem occurs when using a phone(andriod). Here I have to tap twice to get the same effect.

**Comment by melloware** [comment]
Can you open a Bug ticket: https://github.com/primefaces/primefaces/issues

I will fix for 14.0.6 as this should work.

**Reply by YogeshMKumbhar** [reply, marked as answer]
Thanks. Opend an issue https://github.com/primefaces/primefaces/issues/12600
